docs/source/overview/sensors/ray_caster.rst

```diff
@@ -0,0 +1,268 @@
+
+To keep the sensor performant when there are many cloned environments, the line tracing is done directly in `Warp <https://nvidia.github.io/warp/>`_. This is the reason why specific meshes need to be identified to cast against: that mesh data is loaded onto the device by warp when the sensor is initialized. As a consequence, the current iteration of this sensor only works for literally static meshes (meshes that *are not changed from the defaults specified in their USD file*).  This constraint will be removed in future releases.
+
+Using a ray caster sensor requires a **pattern** and a parent xform to be attached to.  The pattern defines how the rays are cast, while the prim properties defines the orientation and position of the sensor (additional offsets can be specified for more exact placement).  Isaac Lab supports a number of ray casting pattern configurations, including a generic LIDAR and grid pattern.
+
+.. code-block:: python
+
+  @configclass
+  class RaycasterSensorSceneCfg(InteractiveSceneCfg):
+      """Design the scene with sensors on the robot."""
+
+      # ground plane with texture for interesting casting results
+      ground = AssetBaseCfg(
+          prim_path="/World/Ground",
+          spawn=sim_utils.UsdFileCfg(
+              usd_path=f"{ISAAC_NUCLEUS_DIR}/Environments/Terrains/rough_plane.usd",
+              scale = (1,1,1),
+          )
+      )
+
+      # lights
+      dome_light = AssetBaseCfg(
+          prim_path="/World/Light", spawn=sim_utils.DomeLightCfg(intensity=3000.0, color=(0.75, 0.75, 0.75))
+      )
+
+      # robot
+      robot = ANYMAL_C_CFG.replace(prim_path="{ENV_REGEX_NS}/Robot")
+
+      ray_caster = RayCasterCfg(
+          prim_path="{ENV_REGEX_NS}/Robot/base/lidar_cage",
+          update_period = 1/60,
+          offset=RayCasterCfg.OffsetCfg(pos=(0,0,0.5)),
+          mesh_prim_paths=["/World/Ground"],
+          pattern_cfg=patterns.LidarPatternCfg(
+            channels = 100,
+            vertical_fov_range=[-90, 90],
+            horizontal_fov_range = [-90,90],
+            horizontal_res=1.0),
+          debug_vis=True,
+      )
+
+
+Notice that the units on the pattern config is in degrees! Also, we enable visualization here to explicitly show the pattern in the rendering, but this is not required and should be disabled for performance tuning. Querying the sensor for data can be done at simulation run time like any other sensor.
+
+.. figure:: ../../_static/overview/overview_sensors_rc_visualizer.png
+    :align: center
+    :figwidth: 100%
+    :alt: Lidar Pattern visualized
+
+Querying the sensor for data can be done at simulation run time like any other sensor.
```

Comment:
this seems to be a repeat of the previous paragraph.

Reply:
ah! yes, i was figuring out where to put the image.  i removed the initial instance in the above paragraph